.github/workflows/regenerate-operator-bundles.yml

```
@@ -113,9 +113,22 @@ jobs:
+        if: ${{ always() }}
+        run: |
+          changes=false
+          echo "::group::Detect Changes in Charts"
+          if ! git diff --exit-code; then
+            changes=true
+          fi
+          echo "::endgroup::"
+          echo "changes=${changes}" >> $GITHUB_OUTPUT
+
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v3
```

actionlint (1.7.12)

[error] the runner of "peter-evans/create-pull-request@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)